codegen/java/java-entities/src/test/groovy/com/stanfy/helium/handler/codegen/java/entity/PojoWriterSpec.groovy

```diff
@@ -538,4 +538,38 @@ public class Test {
 }
 """.trim() + '\n'
   }
+
+  def "custom mapping outside classpath"() {
+    given:
+    Message testMessage = new Message(name: "Test")
+    testMessage.addField(new Field(name: "data", type: new Type(name: "customType")))
+
+    and:
+    options.customPrimitivesMapping = ['customType': 'custom.Name']
```

Comment:
tbh I want to see test with Java generics - something like:
`java.util.Map<String, com.stanfy.CustomClass>`